packages/js/product-editor/src/blocks/category/block.json

```diff
@@ -0,0 +1,31 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-category",
+	"title": "Product Category",
+	"category": "widgets",
+	"description": "A field to select product categories.",
+	"keywords": [ "products", "name", "title" ],
```

Comment:
Should we have category as a keyword? I admittedly don't know how these are being used.

Reply:
Updated, I also removed `name` and `title` as those didn't make much sense.